### PR TITLE
Add Module DocString for Base.Checked

### PR DESCRIPTION
**Pull request by freemin7**
Thanks to Pramodh Gopalan for feedback in Slack.

This should make Checked more approacheable.

### CODE REVIEW

**Comment by dkarrasch** [comment]
Thanks and welcome @freemin7! I believe you have to put the docstring above the line with
```julia
module Checked
```
Second, looking at the `Broadcast` example, I'd put `Base.Checked` in the indented line, followed by a blank line.

**Comment by freemin7** [comment]
Feel free to edit it.

**Comment by dkarrasch** [comment]
Feel free to pick up the advice and finish your first PR. 😉

**Comment by freemin7** [comment]
Ok that was easier than i thought using the GitHub interface

File: base/checked.jl [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 # Support for checked integer arithmetic
 
+"""
+    Checked
```
Comment (dkarrasch) [inline base/checked.jl:6]: There should be a blank line here.

File: base/checked.jl [hunk 2]
```diff
@@ -2,6 +2,13 @@
...
+The Checked module provides arithmetic functions for the built-in signed and unsigned
+Integer types which throw an error when an overflow occurs. They are named like `checked_sub`,
+`checked_div`, etc. In addition, `add_with_overflow, sub_with_overflow, mul_with_overflow`
```
Comment (dkarrasch) [inline base/checked.jl:9]: I think each function should have its own "code style" environment, like
```
`add_with_overflow`, `sub_with_overflow`
```
etc.

**Review by dkarrasch — COMMENTED** [review]
Some minor comments regarding style, otherwise this looks good to me.

**Pull request merged**